tests/cli/archive/test_archive_cli.py

```diff
@@ -0,0 +1,25 @@
+from click.testing import CliRunner
+
+from cg.cli.archive import archive_spring_files
+from cg.models.cg_config import CGConfig
+
+
+def test_limit_and_archive_all_fails(cli_runner: CliRunner, cg_context: CGConfig):
```

Comment:
This test is fine, but maybe we could add a couple more ones that tests a working case?

Reply:
Added tests!